tensorflow/core/kernels/tile_ops_impl.h

```diff
@@ -21,31 +21,8 @@ limitations under the License.
 #include "tensorflow/core/platform/types.h"
```

Comment:
What's the difference between `tile_ops_functor.h` and `tile_ops_impl.h`?  Do they need to be separate files?